Web2palette/manifest.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name" : "Web2Palette",
+    "version": "1.0.0",
+    "version_name": "1.0 dev",
+    "description": "Grab color palette from current tab",
+    "permissions": ["activeTab"],
+    "action": {
+        "default_icon": {
+            "16": "./imgs/icon16.png",
+            "32": "./imgs/icon32.png"
+        },
+        "default_title": "Web2Palette",
+        "default_popup": "pages/popup.html"
+    },
+    "icons": {
+        "16": "./imgs/icon16.png",
+        "48": "./imgs/icon48.png",
+        "128": "./imgs/icon128.png"
+    },
+    "author": "nikhil2003verma@gmail.com",
```

Comment:
change the author to pickapalette

Reply:
Author: it should contain the mail address. It should be the same as the mail used to login to the chrome web store for extension publishing.

Reply:
then I think making it as a env variable will be best. Or Dont commit the manifest file
Write in the readme a sample manifest file and users will modify it according to the choice